client/lib/sites-list/test/index.js

```diff
@@ -85,6 +85,16 @@ describe( 'SitesList', () => {
 				assert.deepEqual( updatedValue, site[ prop ] );
 			} );
 		} );
+
+		it( 'should not trigger change when sites attributes didn\'t change', () => {
+			const changeCallback = sinon.spy();
+
+			sitesList.once( 'change', changeCallback );
+
+			sitesList.update( cloneDeep( data.original ) );
+
+			assert.isFalse( changeCallback.called );
+		} );
```

Comment:
Were you able to get a failing test (with previous changes) without updating the test fixtures? Could we also add one of the computed props to the test data `is_previewable`, `is_customizable`, or slug? The test is 👍 once we do that.